### ISSUE TITLE
Web ui frames scaling

### ISSUE THREAD

**Issue by therb1** [thelastpickle/cassandra-reaper#721]
We have problem with scaling web ui for cluster with big count of nodes in one dc.
What we can do to repair it.

# Browser
```
Google Chrome | 75.0.3770.100 (Официальная сборка) (64 бит) (cohort: Stable)
-- | --
Версия | cd0b15c8b6a4e70c44e27f35c37a4029bad3e3b0-refs/branch-heads/3770@{#1033}
ОС | Windows 10 OS Version 1903 (Build 18362.239)
JavaScript | V8 7.5.288.23
Flash | 32.0.0.223
```
# Image
![Size](https://user-images.githubusercontent.com/9654476/61455507-65bda400-a96c-11e9-82bd-bd939a4ec6d4.PNG)

# Problem
I can see size of information for all nodes. frame bonding.

**Comment by adejanovski** [comment]
Hi @therb1, 

That's currently how Reaper's UI displays the nodes. The size of each node is relative to its data on disk, which will shrink the size of each node below the size of the text in the block. You can hover with the mouse to see the text though.
The intent here is to detect imbalances between nodes easily.

**Comment by therb1** [comment]
Hi @adejanovskiif.
You can scale this block on second row, maybe, you can see imbalance more easily.
![Size](https://user-images.githubusercontent.com/9654476/61498726-15c2f980-a9cd-11e9-9804-df185fa9c206.PNG)

**Comment by adejanovski** [comment]
@therb1,

I don't know how to apply your suggestion using CSS while still keeping a size for each node that is proportional to its load on disk. Furthermore, if you have a cluster with 500 nodes, it will take way too much space on screen.
How did you generate the above screen? Did you set a minimum size for each node? If so, how would this work in case of imbalances?

I'm eager to improve UX but failed to deliver something better so far.

**Comment by therb1** [comment]
It's are gimp image. but i try ask our fronted developers for help on monday.

**Comment by therb1** [comment]
@adejanovski Hi Alexander we fix it.
Can you give me access to create a branch?
that's what we can do.
we change 2 files.

![m o](https://user-images.githubusercontent.com/9654476/62219348-d4602000-b3b6-11e9-88f6-a604c6cffbae.png)

**Comment by adejanovski** [comment]
Hi @therb1,

You lost the relative sizing of the nodes based on their load, which is not something we're willing to give up now.
But your numbers don't add up either it seems, so I'm not sure what your source is for this data.

With large clusters, having 3 nodes per row will not be possible because they will not fit in the screen. With the current UI, I can display clusters with hundreds of nodes in a compact form.

Contributing will be done by forking the project on your github account and then pushing a PR.

**Comment by therb1** [comment]
@adejanovski Can you give me a some screenshots with example? with 100 nodes maybe.
I think i didnt undestend what you mean.
We will emulate your state and fix the layout so that it would be readable.

Could you rephrase your statement about the size of the node relative to its load?
Form is compact but it's a didnt readable in my example.

**Comment by adejanovski** [comment]
This was fixed in 1.4.7 and `master`.
